test/wasm2c/spec/memory64/table.txt

```diff
@@ -1,6 +1,82 @@
 ;;; TOOL: run-spec-wasm2c
-;;; STDIN_FILE: third_party/testsuite/proposals/memory64/table.wast
 ;;; ARGS*: --enable-memory64
+;; memory64 table.wast from last commit before wasm-3.0 merge
```

Comment:
Aren't these files automatically generated by `test/update-spec-tests.py`?    What happens when somebody runs that script again?   Should the script be able to handle this kind of thing?

Reply:
Fortunately the script won't edit a file that already exists (but it will try to create a bunch of other files, e.g. test/wasm2c/spec/memory64/if.txt because all the wasm-3.0 tests are in the memory64 part of the testsuite repo).